app/src/main/java/org/simple/clinic/storage/Migration_13_14.kt

```diff
@@ -0,0 +1,40 @@
+package org.simple.clinic.storage
+
+import android.arch.persistence.db.SupportSQLiteDatabase
+import android.arch.persistence.room.migration.Migration
+import org.simple.clinic.medicalhistory.MedicalHistory
+import org.threeten.bp.Instant
+import java.util.UUID
+
+/**
+ * Adds an empty [MedicalHistory] for all patients.
+ */
+@Suppress("ClassName")
+class Migration_13_14 : Migration(13, 14) {
+
+  override fun migrate(db: SupportSQLiteDatabase) {
```

Comment:
This migration needs to be thoroughly reviewed and tested. Steps to test:

1. Check if there is a patient present on staging that *doesn't* have a medical history stored. This is not the same as a medical history with `None of the above` as the selected answer. If you cannot find any, manually insert one by either running `PatientSyncAndroidTest` or checking out a version of master before medical history was introduced.

2. Run this branch.

3. Open the patient's summary screen

4. The screen shouldn't crash. Scroll to the bottom to find medical history. The history should be empty with no selected answers.